src/protagonist/DLCS.Repository/Messaging/AssetNotificationSender.cs

```diff
@@ -82,7 +97,18 @@ public async Task<bool> SendIngestAssetRequest(Asset assetToIngest, Cancellation
         return statusCode;
     }
 
-    public Task SendAssetModifiedNotification(ChangeType changeType, Asset? before, Asset? after)
+    private async Task<bool> SendCleanupAssetRequest(Asset assetToCleanup, CustomerPathElement customerPathElement, CancellationToken cancellationToken = default)
+    {
+        var notificationRequest = new
```

Comment:
This might come in a later PR but a strongly typed object might make consuming this easier. Producer and (main) consumer are both in this sln so can share the strongly typed model so changes will be easier to handle.

Reply:
yeah, given the next PR has 4 objects required in here, it's probably better to move this to a model class - will get this done in the next PR